src/java.base/share/classes/sun/nio/fs/PollingWatchService.java

```diff
@@ -305,10 +310,10 @@ void enable(Set<? extends WatchEvent.Kind<?>> events, long period) {
                 // update the events
                 this.events = events;
 
-                // create the periodic task
+                // create the periodic task with initialDelay set to the default sensitivity
```

Comment:
I don't think "default sensitivity" is apt any more; perhaps "specified constant" instead?
```// create the periodic task with initialDelay set to the specified constant```

Reply:
:+1: